tests/integration/test_update.py

```diff
@@ -22,10 +24,10 @@ def test_ensemble_parent_child_link(client, simple_ensemble):
     ensemble_0_id = simple_ensemble()
 
     update_ens1_id = _create_dummy_update_id("bogosort", ensemble_0_id, client)
-    ensemble_1_id = simple_ensemble(update_id=update_ens1_id)
+    ensemble_1_id = simple_ensemble(update_id=str(update_ens1_id))
 
     update_ens2_id = _create_dummy_update_id("dijkstra", ensemble_0_id, client)
-    ensemble_2_id = simple_ensemble(update_id=update_ens2_id)
+    ensemble_2_id = simple_ensemble(update_id=str(update_ens2_id))
```

Comment:
Make the test helpers (`create_experiment`, `create_ensemble`, `simple_ensemble`) accept and return `UUID`s rather than strings.